### ISSUE TITLE
Chat history is no longer displayed

### ISSUE THREAD

**Issue by winman3000** [miranda-ng/miranda-ng#3485]
## Details

In Miranda NG, when I open the chat window of a person I have previously chatted with, the chat history is no longer displayed to me. I can observe this behavior at least with a screen reader (JAWS). Whether this is also the case for normally sighted people, I don't know. I use StdMsg and Facebook.

### Steps to reproduce.

1. Make sure you are using StdMsg as the chat module and the Miranda NG is set up to show recent chat history.
1. Open a chat of a person you have chatted with before.
1. Press TAB once to go to the chat history and use the up and down arrow keys to read the history.

#### Result

The chat history is not displayed or cannot be read with a screen reader.

#### Expected

The chat history should be readable using the arrow keys.

**Comment by georgehazan** [comment]
It depends on Options - Message sessions - Messaging log - Load history events
(unread only / number of previous events / less than XXX minutes)

**Comment by georgehazan** [comment]
@winman3000 also I strongly recommend you to switch to the stable version, because there are a lot of major changes in the dev version, and it might be unstable these days

**Comment by winman3000** [comment]
This is all already set up for me, but the new messages I have written are not displayed too.

**Comment by georgehazan** [comment]
no problems, fix will be available at the server in an hour
